whois-api/src/main/java/net/ripe/db/whois/api/httpserver/FixedResponseRule.java

```diff
@@ -0,0 +1,36 @@
+package net.ripe.db.whois.api.httpserver;
+
+import org.eclipse.jetty.rewrite.handler.Rule;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+public class FixedResponseRule extends Rule {
+
+    private final int responseCode;
+
+    public FixedResponseRule(final int responseCode) {
+        this.responseCode = responseCode;
+    }
+
+    @Override
+    public String matchAndApply(String target, HttpServletRequest request, HttpServletResponse response) throws IOException {
+        if (responseCode >= 400) {
```

Comment:
You could avoid the hard-coded status code with (Jetty) `HttpStatus.isClientError(code) || HttpStatus.isServerError(code)`

Reply:
Done.